tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
 	"compilerOptions": {
 		"rootDir": "./",
-		"target": "es2020",
-		"lib": ["dom", "dom.iterable", "esnext"],
+		"target": "ES2017",
```

Comment:
I caused it 🙃 

We had a target which is higher from what most browsers support toady
this might cause issues when consuming it on descoper apps, they will have to transpile our code in order to run it

I even think someone was complaining about it in auth town and you talked with him